### ISSUE TITLE
Description can't be a static field?

### ISSUE THREAD

**Issue by pjf** [ralphbean/bugwarrior#490]
Hello again fabulous bugwarrior devs!

I'm hitting a stumbling block with my workflow, which might just be me misunderstanding the documentation, but it looks like I can't make description into a static field. This is something I'd *really* love, as it means that while bugwarrior can set my task names to begin with, I can modify them to be more descriptive as needed. This is especially useful with the new gmail service, as subject lines don't always reflect the thing I need to be doing.

I have the following config fragment:

```
[general]
static_fields = description
```

But when I run this, bugwarrior-pull gives me:

```
Traceback (most recent call last):
  File "/usr/local/bin/bugwarrior-pull", line 11, in <module>
    load_entry_point('bugwarrior==1.5.1', 'console_scripts', 'bugwarrior-pull')()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 700, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 680, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 873, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 508, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/bugwarrior-1.5.1-py2.7.egg/bugwarrior/command.py", line 73, in pull
    synchronize(issue_generator, config, main_section, dry_run)
  File "/usr/local/lib/python2.7/dist-packages/bugwarrior-1.5.1-py2.7.egg/bugwarrior/db.py", line 351, in synchronize
    del issue_dict[field]
KeyError: u' description'
```

I'm using bugwarrior ddc60f6.

Is there a way to have description be a static field, so bugwarrior can set it on task creation, but not modify it after the task exists? (Right now bugwarrior helpfully reverts my hand-crafted task descriptions back to their defaults whenever it runs.)

Many, many thanks again for an amazing tool!

~ pjf

**Comment by ryneeverett** [comment]
The `issue_dict` ought to have a `description` key at that point in the code. I suspect this is a bug in the gmail service which is somehow failing to add a `description` field.

**Comment by cthulahoops** [comment]
I'm struggling to reproduce this. It seems to just work for me.

I've tried adding the ```description``` to ```static_fields``` before and after fetching the issues initially, with and without a description template.

```GmailService``` doesn't add a description itself, it relies on either ```Issue.refine_record``` to add it either from a template or by calling ```get_default_description```. The only case I see where description wouldn't exist would be if ```description``` didn't appear in ```taskw.task.Task.FIELDS```.

**Comment by cthulahoops** [comment]
Hang on!

```
KeyError: u' description'
```

There's a leading space: the key being accessed is ```" description"``` not ```"description"```. How did that space even get there?

If I set

```
static_fields = tags, description
```

then: ```KeyError: u' description'```

```
bugwarrior/db.py|303 col 9| static_fields = conf.get(main_section, 'static_fields').split(',')
```

Splits on ',' not ', '.

Would this be better as ```re.split(r', *')```?

**Comment by ryneeverett** [comment]
Good eye @cthulahoops and 👍 to your proposal. I'd call this a usability bug.

**Comment by ryneeverett** [comment]
Resolved by #496.
